target-chains/near/receiver/tests/workspaces.rs

```diff
@@ -511,24 +580,27 @@ async fn test_contract_fees() {
         .outcome()
         .is_success());
```

Comment:
Not for this part, rather line 645. Do all transactions in near succeed regardless of their failure in the end? because in some tests I see you commented it should fail and it is still success, then you check the state to figure out.

if so its very weird...